tests/test_cfg.py

```diff
@@ -1,101 +1,98 @@
-from typing import List, Tuple
-import pytest
+import json
+import os
+import sys
+import pathlib
+from pprint import pprint
 
+import pytest
+from deepdiff import DeepDiff
 
-from tealer.teal.basic_blocks import BasicBlock
-from tealer.teal.instructions import instructions
+from tealer.utils.output import bbs_to_json
 from tealer.teal.parse_teal import parse_teal
 
-from tests.utils import cmp_cfg, construct_cfg
-
-
-MULTIPLE_RETSUB = """
-#pragma version 5
-b main
-is_even:
-    int 2
-    %
-    bz return_1
-    int 0
-    retsub
-return_1:
-    int 1
-    retsub
-main:
-    int 4
-    callsub is_even
-    return
-"""
-
-ins_list = [
```

Comment:
I am wondering if we should not keep some tests under this format. It might be useful to build on top more low-level tests. What do you think?

Reply:
yes, it would be better to build in more low-level way. it might be hard to write tests in this way if more tests are needed but it shouldn't be a problem if tests are added incrementally.